src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,36 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver givenDriver){ super(givenDriver); }
+
+    By avatarIcon = By.cssSelector("img.avatar");
+    By username = By.cssSelector("span.name");
+    By allSongs = By.cssSelector("li a.songs");
+    By pauseBtn = By.cssSelector(".pause .fa.fa-pause");
+    By playBtn = By.cssSelector(".play .fa.fa-play");
+    By searchField = By.cssSelector("input[type='search']");
+    By nextSongBtn = By.cssSelector("i[title='Play next song']");
+    By prevSongBtn = By.cssSelector("i[title='Play previous song']");
+    By playlistInputField = By.cssSelector("[name='name']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```